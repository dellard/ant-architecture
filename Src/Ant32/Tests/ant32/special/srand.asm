#tests srand by generating three random numbers, and then setting them to be the new seed
#@ tests srand

	rand 	r1
	rand	r2
	rand	r3

	srand	r1, r2, r3	

