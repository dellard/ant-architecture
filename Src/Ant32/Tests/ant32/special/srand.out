CPU ERROR: An exception occured while exceptions were disabled.
e0 = 0x00000000  e1 = 0x00000000  e2 = 0x80100000  e3 = 0x00000033
     (Shadow PC)      (INT mask)       (TLB addr)       (Exception)

ERROR: Exception occured with exceptions disabled.
